src/main/resources/react4xp/_entries/CookieBanner.tsx

```diff
@@ -57,29 +60,28 @@ function CookieBanner(): JSX.Element | null {
 
   if (!visible) return null
 
+  const cookieLink =
```

Comment:
Maybe this could also be passed a prop so we can use the baseUrl config. The base url is hard coded at the moment.